src/bot/cogs/economy.py

```
@@ -201,75 +327,323 @@ async def inventory_command(self, interaction: discord.Interaction):
+        guild_id = str(interaction.guild_id)
+        user_id = str(interaction.user.id)
+        
+        bet = int(kwargs.get('amount', 100))
+        if bet <= 0:
+            return await interaction.followup.send("1コイン以上を賭けてください。")
+
+        user_data = await self.bot.api.get_economy_user(guild_id, user_id)
+        current_money = user_data.get("money", 0)
+
+        if current_money < bet:
+            return await interaction.followup.send(f"所持金が足りません（所持: {current_money}コイン）。")
```

Ruff (0.15.7)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)